### ISSUE TITLE
Uncaught error: Request cancelled.

### ISSUE THREAD

**Issue by phyllisstein** [atom/ide-typescript#102]
Hey folks! I'm having some trouble with the TypeScript IDE on Atom `v1.25.0` and `v1.26.0-beta0`. Completions from the TypeScript server take ≈10s to display after I stop typing (à la #1), and the IDE package logs a slew of errors to the console:

```
TypeScript (SourceGraph) stderr WARN  $/cancelRequest for unknown request ID 118

Uncaught (in promise) Error: Request cancelled
    at new ResponseError (/Users/daniel/.atom/packages/ide-typescript/node_modules/vscode-jsonrpc/lib/messages.js:46)
    at handleResponse (/Users/daniel/.atom/packages/ide-typescript/node_modules/vscode-jsonrpc/lib/main.js:430)
    at processMessageQueue (/Users/daniel/.atom/packages/ide-typescript/node_modules/vscode-jsonrpc/lib/main.js:258)
    at Immediate.<anonymous> (/Users/daniel/.atom/packages/ide-typescript/node_modules/vscode-jsonrpc/lib/main.js:242)
    at runCallback (timers.js:672)
    at tryOnImmediate (timers.js:645)
    at processImmediate [as _immediateCallback] (timers.js:617)
```

I made a quick screencast demonstrating the issue:
![screenflow](https://user-images.githubusercontent.com/2207980/38146181-dfc4e984-341a-11e8-8fa4-ee4a4b5b3c87.gif)

It seems to be pretty consistent---and it may be affecting other IDE packages as well, as I had to uninstall the Ruby and Elixir IDEs to get the error to appear in the right place. If you have any thoughts on what might be going on I'd be glad to hear them; happy to furnish more debugging color if I can.

**Comment by sbrl** [comment]
Yep, I'm seeing this too:

![](https://i.imgur.com/BySEuji.png)

This makes the package unusable. What's the status on this? This package is basically the only one available for Javascript linting etc., which makes it extremely valuable. I assume it is dead?

**Comment by damieng** [comment]
Should be fixed with the new ide-typescript 0.8.0 update.

**Comment by phyllisstein** [comment]
That's awesome! Thank you for clearing this up.
